### DISCUSSION TITLE
Sync favorites and notes

### DISCUSSION THREAD

**Discussion by guettli** [osmandapp/OsmAnd#11490] [General]
AFAIK all my favorites and notes will be gone if my phone will get lost, since this data is stored only on the phone and not in the cloud.

My list of favorites is getting longer and I feel fear that it could get lost.

How to sync my data so that it does not get lost?

**Comment by pebogufi** [comment]
favourites.gpx is a FILE in the same directory, where the maps are.
So you can copy it or share with a file manager. That's so easy.

**Reply by guettli** [reply]
I am looking for an automated solution which syncs my data daily. I believe you, that it is easy for you. But I guess it is hard for most people.
